home-reader/src/main/java/com/softserveinc/ita/homeproject/readerapp/controllers/ApartmentApiReaderImpl.java

```diff
@@ -0,0 +1,31 @@
+package com.softserveinc.ita.homeproject.readerapp.controllers;
+
+import java.util.List;
+
+import com.softserveinc.ita.homeproject.readerapp.models.ApartmentReader;
+import com.softserveinc.ita.homeproject.readerapp.service.ApartmentReaderService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class ApartmentApiReaderImpl {
```

Comment:
consider not using "reader" word in all classes, since the package already tells it's reader + its shorter